code/modules/atmospherics/environmental/LINDA_fire.dm

```diff
@@ -14,11 +14,14 @@
 	if(!air_gases)
 		return
 
-	var/oxy = air_gases[/datum/gas/oxygen] ? air_gases[/datum/gas/oxygen][MOLES] : 0
+	. = air_gases[/datum/gas/oxygen]
+	var/oxy = . ? .[MOLES] : 0
 	if (oxy < 0.5)
 		return
-	var/tox = air_gases[/datum/gas/plasma] ? air_gases[/datum/gas/plasma][MOLES] : 0
-	var/trit = air_gases[/datum/gas/tritium] ? air_gases[/datum/gas/tritium][MOLES] : 0
+	. = air_gases[/datum/gas/plasma]
+	var/tox = . ? .[MOLES] : 0
```

Comment:
( . ? . )

Reply:
this looks spooky.

Reply:
this will work as intended but it's not very clean